test/bats/aws.bats

```diff
@@ -7,34 +7,46 @@ SLEEP_TIME=1
 PROVIDER_YAML=https://raw.githubusercontent.com/aws/secrets-store-csi-driver-provider-aws/main/deployment/aws-provider-installer.yaml
 NAMESPACE=kube-system
 POD_NAME=basic-test-mount
-export REGION=us-west-2
+export REGION=$REGION
```

Comment:
nit:

How about `export REGION=${REGION:-us-west-2}`. By doing this we don't need to pass the `REGION` in the `e2e-aws` target.